crates/pop-cli/src/commands/new/parachain.rs

```diff
@@ -129,7 +132,7 @@ async fn guide_user_to_generate_parachain() -> Result<NewParachainCommand> {
 		decimals: 12,
 		initial_endowment: "1u64 << 60".to_string(),
 	};
-	if template.matches(&Provider::Pop) {
+	if Provider::Pop.matches(&template) {
```

Comment:
Note, before individual templates were able to use the method `matches` to see if it belongs to a TemplateType; "does template type match template". This PR has flipped this functionality and now matches the template to the template type; "does this template match the template type".